core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -71,9 +71,9 @@
   private final ConcurrentHashMapV8<SocketAddress, NettyChannelPool>
       mNettyChannelPools = new ConcurrentHashMapV8<>();
 
-  /** The shared master address associated with the {@link FileSystemContext}. */
+  /** The shared master inquire client associated with the {@link FileSystemContext}. */
   @GuardedBy("this")
```

Comment:
The former. The client can be used concurrently - I've added `@ThreadSafe` to the `MasterInquireClient` interface.